great_expectations/rule_based_profiler/types/data_assistant_result/plot_components.py

```diff
@@ -153,13 +154,19 @@ def determine_plot_title(
         An Altair TitleParam object
 
     """
-    contents: str = f"{metric_plot_component.title} per {batch_plot_component.title}"
+    contents: str
+    if expectation_type:
+        contents = expectation_type
+    else:
+        contents: str = (
+            f"{metric_plot_component.title} per {batch_plot_component.title}"
+        )
     subtitle: Optional[str] = domain_plot_component.subtitle
     domain_selector: Optional[str] = domain_plot_component.name
 
     title: alt.TitleParams
     if subtitle:
-        title = alt.TitleParams(contents, subtitle=[subtitle])
+        title = alt.TitleParams(contents, subtitle=[subtitle], dy=-25)
```

Comment:
Is this still needed now that we don't have the subtitle + domain selector case? If you look at the return charts I'm guessing they are going to have too much space somewhere in the title/subtitle.

Reply:
fixed!